Gemfile.lock

```diff
@@ -0,0 +1,21 @@
+PATH
```

Comment:
Some projects that have complicated package dependency, like to manage `Gemfile.lock`.
But simple project like this project does not like to manage `Gemfile.lock`.
I think that this file should be included in `.gitignore`.